docs/release-notes/8.16.asciidoc

```diff
@@ -0,0 +1,157 @@
+* Ensures that {elastic-defend} tells {fleet} that it's `orphaned` if the connection between {elastic-defend} and {agent} stops for an extended period of time. {fleet} uses this information to provide you with additional troubleshooting context.
+* Adds SOCKS5 proxy support to {elastic-defend}'s {ls} output.
+* Ensures that on Windows, {elastic-defend} uses https://www.elastic.co/security-labs/finding-truth-in-the-shadows[Intel CET and AMD Shadow Stacks] to collect call stacks, where supported. This improves performance and enables the detection of certain defense evasions. You can turn this feature off in {elastic-defend} <<adv-policy-settings,advanced policy settings>> ({kibana-pull}190553[#190553]).
+* Restore {elastic-defend}'s support for Windows Server 2012, which was removed in 8.13.0.
+* Improves {elastic-defend}'s caching to reduce memory usage on Windows.
+* Enhances {elastic-defend} by reducing the size of process events, which reduces excessive process ancestry entries and shortens the entity ID.
+* Improves the reliability and system resource usage of {elastic-defend}'s Windows network driver.
+
+[discrete]
+[[bug-fixes-8.16.0]]
+==== Bug fixes
+
+* Prevents an empty warning message from appearing for rule executions ({kibana-pull}186096[#186096]). 
+* Fixes an error that could occur during rule execution when the source index had a non-ECS-compliant text field ({kibana-pull}187673[#187673]). 
+* Fixes an issue that could cause fields for all indices to appear when you tried to add a rule filter ({kibana-pull}194678[#194678]).
+* Removes unnecessary empty space below the title of the Open Timeline modal ({kibana-pull}188837[#188837]). 
+* Improves the performance of the Alerts table ({kibana-pull}192827[#192827]). 
+* Removes the requirement that you have unnecessary {kib} {fleet} privileges to access some cloud security posture findings ({kibana-pull}194069[#194069]).
+* Fixes an {elastic-defend} bug where network event deduplication logic could incorrectly drop Linux network events.
+* Fixes an {elastic-defend} bug where Windows API events might be dropped if they contain Unicode characters that can't be converted to ANSI.
+* Ensures that {elastic-defend} does not emit an empty `memory_region` if it can't enrich a memory region in an API event. With this fix, {elastic-defend} removes these fields.
+* Fixes a bug where {elastic-defend} could fail to properly enrich Windows API events for short-lived processes on older operating systems that don't natively include this telemetry, such as Windows Server 2019. This might result in dropped or unattributed API events. 
+* Fixes a bug that prevented host name uniformity with {beats} products. If you request for {elastic-defend} to use the fully qualified domain name (FQDN) in the `host.name` field, {elastic-defend} now reports the FQDN exactly as the OS reports it, instead of lowercasing by default.
```

Comment:
```suggestion
* Fixes a bug that prevented host name uniformity with {beats} products. If you request {elastic-defend} to use the fully qualified domain name (FQDN) in the `host.name` field, {elastic-defend} now reports the FQDN exactly as the OS reports it, instead of lowercasing by default.
```